src/main/java/dev/aziz/vehiclesfinder/dtos/VehicleSummaryDto.java

```diff
@@ -0,0 +1,22 @@
+package dev.aziz.vehiclesfinder.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class VehicleSummaryDto {
+
+    private Long id;
+    private String brand;
+    private String model;
+    private Integer production_year;
```

Comment:
In java, we don't use the snake case naming strategy. We use to camel case naming strategy. It should be `productionYear`.